noir/noir-repo/acvm-repo/acir/src/circuit/opcodes.rs

```diff
@@ -18,6 +21,7 @@ pub enum Opcode {
     BlackBoxFuncCall(BlackBoxFuncCall),
     Directive(Directive),
     Brillig(Brillig),
+    BrilligPointer(BrilligPointer),
```

Comment:
```suggestion
    BrilligCall(BrilligPointer),
```

I'm thinking we should avoid the `BrilligPointer` terminology and just have it as a `BrilligCall` to be similar to the `Call` for other ACIR functions.

Reply:
At this point there's a question in my mind on why we have a separate `BrilligPointer` struct here but not for ACIR calls, is this necessary?

Reply:
> At this point there's a question in my mind on why we have a separate `BrilligPointer` struct here but not for ACIR calls, is this necessary?

I was debating the same thing with regards to `BrilligCall` and the separate struct happy to make the switch. I was just keeping with the existing `Brillig` file/struct layout, but we can switch if we so desire.

Reply:
Yeah I think we can aim for equivalence with the standard `Call` considering we're going to be removing much of the contents of the `Brillig` file.